apps/prairielearn/src/lib/comments.ts

```diff
@@ -0,0 +1,11 @@
+export function isRenderableComment(comment) {
```

Comment:
Nit, we can add some type here.

```suggestion
export function isRenderableComment(comment: string | any[] | Record<string, any>) {
```

Reply:
Added types in [2e19053](https://github.com/PrairieLearn/PrairieLearn/pull/11947/commits/2e19053eb792d55be58d106aedfa6a9ba6020601).